Modify rule S2761: LaYC multiple unary operators (#2180)
SonarSource/rspec: rules/S2761/vbnet/rule.adoc

== Why is this an issue?

> Calling the  ``++Not++`` operator twice is usually a typo. The second invocation invalidates the first one:

Reviewer: ```suggestion
The repetition of the  ``++Not++`` operator is usually a typo. The second operator invalidates the first one:
```

== Why is this an issue?

> Calling the ``++Not++`` operator twice does nothing: the second invocation undoes the first. Such repetitions are usually typos.

Reviewer: ```suggestion
Calling the  ``++Not++`` operator twice is usually a typo. The second invocation invalidates the first one:
```

[source,vbnet]
----
Dim b As Boolean = False
Dim c As Boolean = Not Not b 'Noncompliant: same as b
----

> Either this is a bug if the operator was meant to be called once, or misleading if done on purpose.

Reviewer: ```suggestion
```
If we want to keep the idea, maybe change the first sentence to something like:
`Calling the  ``++Not++`` operator twice is either a typo or misleading because the second invocation invalidates the first one:`
(and then do the same for all the other languages)
